### PR TITLE
US15

### PR DESCRIPTION
**Pull request by Daniz2000**
Contact feature is done

### CODE REVIEW

**Comment by evabaldwin** [comment]
Looks good Mahdi, I'll let Hamid to review it also

**Review by Oleksandrozhoh — APPROVED** [review]

File: src/test/resources/features/US15_Contact.feature [hunk 1]
```diff
@@ -0,0 +1,12 @@
+
+
+Feature: As a user, I should be able to access to Contacts module.
+  As a user, I should be able to access to Contacts module.
+   Background:
+     Given user is on the "Dashboard" page
```
Comment (Oleksandrozhoh) [inline src/test/resources/features/US15_Contact.feature:6]: Excellent collaboration guys, here we see how Mehdi using the parameterized step, that was suggested by Eva during the meeting. Without the parameterization it wouldnt be possible

File: src/test/java/net/meetsky/runners/AlulaRunner.java [hunk 2]
```diff
@@ -17,7 +17,7 @@
                 "json:target/cucumber.json"
         },
         publish = true,
-        tags = "",
+        tags = "@wip",
```
Comment (hamidalakbarov) [inline src/test/java/net/meetsky/runners/AlulaRunner.java:20]: Mehdi, could you please remove @wip tag here and push one more time? Thanks

File: src/test/resources/features/US15_Contact.feature [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+ @wip
```
Comment (hamidalakbarov) [inline src/test/resources/features/US15_Contact.feature:7]: Also from here too, thanks!

**Comment by hamidalakbarov** [comment]
Mahdi, also please make sure your branch name follows the Naming Convention:

**Review by hamidalakbarov — APPROVED** [review]
Looks good Mahdi!

**Pull request merged**